code/modules/shuttle/ert.dm

```diff
@@ -88,17 +88,6 @@
 /obj/machinery/computer/shuttle/ert
 	interaction_flags = INTERACT_MACHINE_TGUI|INTERACT_MACHINE_NOSILICON //No AIs allowed
 
-/obj/machinery/computer/shuttle/ert/valid_destinations()
-	var/obj/docking_port/mobile/ert/M = SSshuttle.getShuttle(shuttleId)
-	if(!istype(M))
```

Comment:
?

Reply:
I don't exactly remember why but I believe this proc was redundant. The way ERT shuttles work is a mess. They used a list of strings with the names of docking ports so they needed to be validated.